File: src/main/java/com/now/nowbot/model/JsonData/SearchCursor.java
package com.now.nowbot.model.JsonData;

import com.fasterxml.jackson.annotation.JsonAutoDetect;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
public class SearchCursor {
    @JsonProperty("queued_at")
    String queued;
    @JsonProperty("approved_data")
    String approved;
    @JsonProperty("id")
    Long id;
}

File: src/main/java/com/now/nowbot/model/osu/SearchCursor.kt
package com.now.nowbot.model.osu

import com.fasterxml.jackson.annotation.JsonAutoDetect
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import tools.jackson.databind.PropertyNamingStrategies
import tools.jackson.databind.annotation.JsonNaming

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
class SearchCursor {
    // @JsonProperty("queued_at") var queued: String? = null

    @JsonProperty("approved_date") var approvedDate: String? = null

    @JsonProperty("id") var id: Long? = null
}
